jobs/io/ndb_io.py

```diff
@@ -74,32 +70,21 @@ def expand(self, pbegin):
 class PutModels(beam.PTransform):
     """Writes NDB models to the datastore."""
 
-    def __init__(self, datastoreio_stub, label=None):
-        """Initializes the PutModels PTransform.
-
-        Args:
-            datastoreio_stub: stub_io.DatastoreioStub. The stub instance
-                responsible for handling datastoreio operations.
-            label: str|None. The label of the PTransform.
-        """
-        super(PutModels, self).__init__(label=label)
-        self.datastoreio = datastoreio_stub
-
-    def expand(self, model_pcoll):
+    def expand(self, input_or_inputs):
         """Writes the given models to the datastore.
 
         Args:
-            model_pcoll: PCollection. A PCollection of NDB models.
+            input_or_inputs: PCollection. A PCollection of NDB models.
```

Comment:
Try to relate these args/returns to the function's name. E.g. -- A PCollection of NDB models to write to the datastore.

Also not clear to me what the point of the return value is?

Finally, if this is an overridden method, please explain that clearly in the docstring.

Reply:
@brianrodri PTAL

Reply:
Done.